src/lib/Browser.js

```diff
@@ -364,6 +364,15 @@ class Browser {
             }
         };
     }
+
+    /**
+     * Returns window hostname
+     *
+     * @return {string} Window hostname
+     */
+    static getHostname() {
```

Comment:
This seems a bit out of place in this file to me. There is a few instances of window.location in the codebase, maybe create a Url class and we can refactor?

Reply:
If there's nothing else needed for URL related stuff, maybe just add it to util since it's being used in a few places?

Reply:
I specifically moved this here (instead of including in util.js) so I could test `isBoxWebApp()` without refactoring util to a class with all static functions or adding Babel Rewire. I'll move this to a new URL class per Daniel's recommendation.